dashboard/test/ui/features/foundations/hamburger.feature

```diff
@@ -1,13 +1,104 @@
-@no_mobile
 @single_session
 Feature: Hamburger dropdown
 
+  @eyes_mobile
+  Scenario: Signed-out user sees the hamburger on mobile
```

Comment:
What does this test do that isn't in the next test? It seems like they're both for @eyes_mobile

Reply:
So this test in particular checks the hamburger nav while the test right after tests the dropdowns within the the hamburger nav. We want a baseline for the hamburger nav without any of the entries expanded as when you expand the Teach section for example, some of the other entries are not visible.

Reply:
I could also combine the two tests?

Reply:
Gotcha that makes sense! No need for a change!